src/main/java/redis/clients/jedis/JedisClusterConnectionHandler.java

```diff
@@ -62,9 +60,7 @@ public void renewSlotCache() {
       } catch (JedisConnectionException e) {
         // try next nodes
       } finally {
-        if (jedis != null) {
```

Comment:
Jedis is initialized as null here: https://github.com/xetorthio/jedis/pull/1303/files#diff-36ffb2068ca7e0edd549920989880b0eR55 if the try bluck throws an exception we need to do this check here.